### PR TITLE
Implement Kendo SASS themes for the DropDownTree widget

### PR DESCRIPTION
**Pull request by mvgmvg**
Implement Default, Bootstrap, and Material to DropDownTree widget

related https://github.com/telerik/kendo/issues/8252

DropDownTree widget uses k-multiselect-wrap class name in its rendering for the multiple selection mode.
To inherit styles from the Multiselect widget, I have updated its selectors as well.

### CODE REVIEW

File: packages/bootstrap/scss/multiselect/_theme.scss [hunk 1]
```diff
@@ -1,28 +1,25 @@
...
         transition: $transition;
 
+        &:hover,
         &.k-state-hover {
             @include appearance( hovered-input );
         }
 
+        &:focus,
         &.k-state-focused {
             .k-multiselect-wrap {
```
Comment (gyoshev) [inline packages/bootstrap/scss/multiselect/_theme.scss]: This rule searches for an element that is `.k-multiselect-wrap.k-state-focused > .k-multiselect-wrap`, which doesn't exist, and breaks the focused state:

image

File: packages/material/scss/popup/_layout.scss [hunk 2]
```diff
@@ -3,6 +3,8 @@ $popup-header-padding: $padding-y-lg $padding-x-lg !default;
 $popup-header-font-size: $font-size-sm !default;
 $popup-header-text: rgba($base-text, .54) !default;
 $popup-filter-padding-x: $padding-x;
+$dropdowntree-popup-spacer-y: 0 !default;
+$dropdowntree-popup-spacer-x: $popup-padding-y !default;
```
Comment (gyoshev) [inline packages/material/scss/popup/_layout.scss]: These variables are for the dropdowntree, and they should not be in the popup files.

File: packages/bootstrap/scss/popup/_layout.scss [hunk 3]
```diff
@@ -1 +1,3 @@
-@import "~@progress/kendo-theme-default/scss/popup/layout";
\ No newline at end of file
+$dropdowntree-popup-spacer-y: 0 !default;
```
Comment (gyoshev) [inline packages/bootstrap/scss/popup/_layout.scss]: This variable is for the dropdowntree, and it should not be in the popup file.

File: packages/default/scss/dropdowntree.scss [hunk 4]
```diff
@@ -0,0 +1,12 @@
+@import "variables";
+@import "mixins";
+
+
+// Dependencies
+@import "common/all";
+@import 'common/forms';
+
```
Comment (gyoshev) [inline packages/default/scss/dropdowntree.scss]: It seems that the dropdowntree depends on the popup, so this should be listed as a dependency.

File: packages/default/scss/popup/_layout.scss [hunk 5]
```diff
@@ -3,6 +3,8 @@ $popup-header-padding: $list-item-padding-y $list-item-padding-x !default;
 $popup-header-font-size: null !default;
 $popup-header-text: null !default;
 $popup-filter-padding-x: false !default;
+$dropdowntree-popup-spacer-x: $padding-y-lg !default;
+$dropdowntree-popup-spacer-y: $padding-y-lg !default;
```
[the remark this answers is not in the document]
Comment (mvgmvg) [reply packages/default/scss/popup/_layout.scss]: I have moved it to the correct file. Actually, regarding the design dropdowntree popup spacer should use $padding-y-lg in both directions.

File: tests/visual/dropdowns.html [hunk 6]
```diff
@@ -2,17 +2,19 @@
 <html lang="en">
 <head>
     <title>Dropdowns</title>
-    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="../../packages/bootstrap/dist/all.css" />
```
[the remark this answers is not in the document]
Comment (mvgmvg) [reply tests/visual/dropdowns.html]: Cannot reproduce, probably it had been fixed. The provided solution is appropriate maybe for one-time load testing.

**Pull request merged**